### PR TITLE
Release scripts

### PR DESCRIPTION
**Pull request by jstucke**
- added release scripts and dependencies
- updated metadata

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,10 +9,20 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "entropython"
 version = "1.1.0"
+readme = "README.md"
+description = "A Python module for efficient calculation of Shannon byte entropy based on Rust"
```
Comment (vobst) [inline pyproject.toml]: Add a full stop?

File: _build-wheels.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -ex
```
Comment (vobst) [inline _build-wheels.sh]: While we are at it, I also tend to set `-u` (just for good practice, not going to change anything here). Helped me quite often.

**Review by vobst — CHANGES_REQUESTED** [review]
Looks good, just some minor nits.

File: _build-wheels.sh (hunk 2 above)
Comment (jstucke) [reply _build-wheels.sh]: done.

**Review by vobst — APPROVED** [review]
Looks good - thanks! :)

**Pull request merged**